tweets/2020/01/wombat-dressing-room.tweet

```diff
@@ -0,0 +1,6 @@
+Wombat Dressing Room, a 2FA proxy for npm 🎉 
+
+automate publishing to npm, while keeping 2FA enabled; from @SemanticRelease'
+friends at @GoogleCloud.
```

Comment:
as this tweet will be sent from `@SemanticRelease`, I would maybe rephrase it to 

> automate publishing to npm, while keeping 2FA enabled; from our friends at @GoogleCloud

I'd also remove the line break